openedx/frontend-app-learner-dashboard
# fix: display programs nav button only if it is configured

**Description**

This PR removes the link of programs from the Header if the service is not configured.

It aims to solve the following issue https://github.com/openedx/frontend-app-learner-dashboard/issues/372

**How to test**

   1. Create a instance with frontend-app-learner-dashobord mounted in the current branch.
   1. Launch the instance and open it in the browser.
   1. Access the learner dashboard.
   1. By default you should not see nav button.
